docs/index.rst

```diff
@@ -56,6 +56,7 @@ conda `environment.yml <https://github.com/SymbiFlow/sphinxcontrib-hdl-diagrams/
 
 - `yosys <https://github.com/YosysHQ/yosys>`_ (required)
 - `netlistsvg <https://github.com/nturley/netlistsvg>`_ (optional)
+- `GHDL <https://github.com/ghdl/ghdl>`_ (required for VHDL)
```

Comment:
I would add a link to the ghdl-yosys-plugin repo too. Strictly, GHDL alone is useless for this extension.